pkg/gluster-virtblock/Dockerfile

```diff
@@ -58,10 +58,11 @@ RUN ldd /build/glustervirtblock-csi-driver | grep -q "not a dynamic executable"
 
 #-- Final container
 
-FROM docker.io/centos:7.5.1804 as final
+FROM docker.io/centos:7 as final
 
 # Install dependencies
-RUN yum -y install centos-release-gluster && \
+RUN yum update -y && \
+    yum -y install centos-release-gluster && \
```

Comment:
@JohnStrunkI think we are missing `ADD http://artifacts.ci.centos.org/gluster/nightly/master.repo  /etc/yum.repos.d/glusterfs-nightly.repo` in this dockerfile  can you  add  it?

Reply:
done

Reply:
@JohnStrunk do we need `centos-release-gluster` or `glusterfs-fuse` here?

Reply:
@kotreshhr why `centos-release-gluster` is added here?

Reply:
We probably just need the fuse driver like the glusterfs driver container